Makefile

```diff
@@ -89,7 +89,21 @@ __check_var = \
     $(if $(value $1),, \
       $(error Undefined variable $1$(if $2, ($2))))
 
+check-and-create-namespace:
```

Comment:
I'm a bit confused -- VPCRC should install into `kube-system`. Wondering why your namespace is missing the `kube-` prefix.

Reply:
@ellistarn, the ns system is for the cert-manager certificate https://github.com/aws/amazon-vpc-resource-controller-k8s/blob/master/config/certmanager/certificate.yaml#L9